paparazzi-gradle-plugin/src/test/java/app/cash/paparazzi/gradle/PaparazziPluginTest.kt

```diff
@@ -759,7 +759,11 @@ class PaparazziPluginTest {
       "app.cash.paparazzi.plugin.test.module1",
       "app.cash.paparazzi.plugin.test.module2"
     )
-    assertThat(config.projectResourceDirs).containsExactly("src/main/res", "src/debug/res")
+    assertThat(config.projectResourceDirs).containsExactly(
+      "src/main/res",
+      "src/debug/res",
+      "build/intermediates/packaged_res/debug"
```

Comment:
Interesting idea pulling this, looking at how AS handles it we could also do something similar in our `LayoutPullParser`

https://cs.android.com/android-studio/platform/tools/adt/idea/+/mirror-goog-studio-main:rendering/src/com/android/tools/rendering/parsers/LayoutRenderPullParser.java;l=972

https://cs.android.com/android-studio/platform/tools/adt/idea/+/mirror-goog-studio-main:android/rendering/testSrc/com/android/tools/idea/rendering/parsers/LayoutRenderPullParserTest.java;l=492?q=AaptAttr

Reply:
agreed.  I noticed similar, but I think our current AaptParser logic might need some refactoring.  I'm exploring that now.

Reply:
following up on this: I debugged how LayoutLib renders `fragment_dashboard.xml` (i.e., the repro from https://github.com/cashapp/paparazzi/issues/1016#issuecomment-1646342203) in the IDE and this layout:

```
<?xml version="1.0" encoding="utf-8"?>
<layout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    xmlns:tools="http://schemas.android.com/tools">

    <androidx.constraintlayout.widget.ConstraintLayout
        android:layout_width="match_parent"
        android:layout_height="match_parent"
        tools:context=".ui.dashboard.DashboardFragment">

        <TextView
            android:id="@+id/text_dashboard"
            ... />
    </androidx.constraintlayout.widget.ConstraintLayout>

</layout>
```

results in this TagSnapshot hierarchy:

```
TagSnapshot{
  androidx.constraintlayout.widget.ConstraintLayout, 
  attributes=[
    AttributeSnapshot{layout_width="match_parent"}, 
    AttributeSnapshot{layout_height="match_parent"}, 
    AttributeSnapshot{context=".ui.dashboard.DashboardFragment"}, 
    AttributeSnapshot{tag="layout/fragment_dashboard_0"}
  ], 
  children=[
    TagSnapshot{
      TextView, 
        attributes=[
          AttributeSnapshot{id="@+id/text_dashboard"}, 
          ...
	], 
	children=[]
     }
  ]
}
```

In particular, note the `AttributeSnapshot{tag="layout/fragment_dashboard_0"}`, which reminds me of the work we did in https://github.com/cashapp/paparazzi/pull/196 to support Cash Card gradients.